### ISSUE TITLE
Create recipe / Troubleshooting Art of Problem Solving 

### ISSUE THREAD

**Issue by RavanJAltaie** [openzim/zim-requests#1473]
As per this https://github.com/openzim/zim-requests/issues/763, please check why this recipe is failing while we have already a successful resulted file in the past.

The link to the recipe on the Zimfarm:
https://farm.openzim.org/recipes/art_of_problem_solving

The link to last working ZIM (if any):
https://download.kiwix.org/zim/other/artofproblemsolving_en_all_maxi_2021-03.zim

The error I get:
Unable to find appropriate API endpoint to retrieve article HTML

**Comment by kelson42** [comment]
Actually we have no recipe for this, we should first create one and see how MWoffliner behaves. Online wiki address is artofproblemsolving.com/wiki/. Moving to `openzim/zim-requests` for the moment as I strongly suspect it works fine now.

**Comment by benoit74** [comment]
Recipe moved here: https://farm.openzim.org/recipes/art-of-problem-solving_en_all

**Comment by inconspicuous-dev** [comment]
The site became ultra slow, crawling might be an issue if the zim crawler's time out time is too short. Note for if the trials fail for that reason.

(I'm not apart of the zim team, just want a zim because the site is slow and the most recent version on internet archive is from may 2025)

**Comment by benoit74** [comment]
Indeed, we cannot scrape such a slow wiki, especially since it will probably only make the situation worse for most users.

**Comment by inconspicuous-dev** [comment]
The weird thing is after I tried independently scraping the wiki to create a zim, it became a bit faster. 

(Probably due to site cache)
Site doesn't get slower after crawling, usually it stays the same.
Although the load times are still abysmally slow and the webcrawler will most likely time out.

**Comment by Popolechien** [comment]
So what's our next step here? Close as `WONTFIX`?

**Comment by benoit74** [comment]
Site is not slow anymore, we should give it a new try (recipe is at https://farm.openzim.org/recipes/art-of-problem-solving_en_all now)
